gulp-tasks/test.js

```diff
@@ -502,6 +503,19 @@ function testMarkdown(filename, contents, options) {
       });
     }
 
+    // Validate listed components
+    do {
+      matched = wfRegEx.RE_BLINK_COMPONENTS.exec(contents);
+      if (matched && options.blinkComponents) {
+        position = {line: getLineNumber(contents, matched.index)};
+        let component = matched[1].trim();
+        if (options.blinkComponents.indexOf(component) === -1) {
+          msg = `The component (\`${component}\`) is non-standard or misspelled.`;
+          logWarning(filename, position, msg);
```

Comment:
This should be an error if it doesn't match. We don't want to accidentally let warnings through.